src/go/rpk/pkg/adminapi/api_debug.go

```diff
@@ -131,6 +135,9 @@ type DebugPartition struct {
 	Replicas []ReplicaState `json:"replicas"`
 }
 
+// CPUProfile is a CPU profile obtained from the Redpanda profiler.
+type CPUProfile map[string]any
```

Comment:
This type is unnecessary right?

Reply:
:sweat_smile: Yeah, we were discussing this and I forgot to delete it. Thanks